### PR TITLE
Small update for the converter

### PR DESCRIPTION
**Pull request by nburmaso**
*configurable TFNumber 
*collision time wrt to the beginning of collision BC

### CODE REVIEW

File: Detectors/AOD/src/AODProducerWorkflowSpec.cxx [hunk 1]
```diff
@@ -478,6 +482,8 @@ void AODProducerWorkflowDPL::run(ProcessingContext& pc)
       firstVtxGlBC = globalBC;
     }
 
+    // collision timestamp in ns wrt to the beginning of collision BC
+    tsTimeStamp = tsTimeStamp - std::lround(tsTimeStamp / o2::constants::lhc::LHCBunchSpacingNS) * o2::constants::lhc::LHCBunchSpacingNS;
```
Comment (jgrosseo) [inline Detectors/AOD/src/AODProducerWorkflowSpec.cxx]: Is this considering if the most probably BC which is filled into GlobalBC is off by 2 BCs ?

Comment (nburmaso) [reply Detectors/AOD/src/AODProducerWorkflowSpec.cxx]: No, I think I should correct this to 
`tsTimeStamp = globalBC * o2::constants::lhc::LHCBunchSpacingNS - tsTimeStamp`
The code currently does not account for a filling scheme, so most probable BC == meanBC

Comment (jgrosseo) [reply Detectors/AOD/src/AODProducerWorkflowSpec.cxx]: Did Ruben already put the function in the vertex? To get the most probably BC moved by the filling scheme? I think it should be on his side, not yours. Then you just take that value.

Comment (nburmaso) [reply Detectors/AOD/src/AODProducerWorkflowSpec.cxx]: As far as I know, it is not ready yet. At least I do not see the function in [PrimaryVertex.h](https://github.com/AliceO2Group/AliceO2/blob/dev/DataFormats/Reconstruction/include/ReconstructionDataFormats/PrimaryVertex.h) or in [Vertex.h](https://github.com/AliceO2Group/AliceO2/blob/dev/DataFormats/Reconstruction/include/ReconstructionDataFormats/Vertex.h) (if it even should be in these classes).

**Review by jgrosseo — APPROVED** [review]

**Pull request merged**